src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -235,6 +241,13 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
    */
   _onError(reject: Function, error: FakeEvent): void {
     NativeAdapter._logger.error(error);
+    const message = PlayerError.createError({
```

Comment:
better handle the error dispatching in html5, and only use this for the load promise rejecting